components/tooltip/index.en-US.md

```diff
@@ -58,9 +59,11 @@ This is due to the implementation of `rc-trigger`. `rc-trigger` forces children
 - `findDOMNode is deprecated` reproduce: <https://codesandbox.io/p/sandbox/finddomnode-c5hy96>
 - Using `forwardRef` to fix: <https://codesandbox.io/p/sandbox/no-finddomnode-warning-forked-gdxczs>
 
-### Why sometime not work on HOC?
+#### Why is the tooltip for my custom component not opening?
 
-Please ensure that the child node of `Tooltip` accepts `onMouseEnter`, `onMouseLeave`, `onPointerEnter`, `onPointerLeave`, `onFocus`, `onClick` events.
+The `Tooltip` component requires the child node to accept `onMouseEnter`, `onMouseLeave`, `onPointerEnter`, `onPointerLeave`, `onFocus`, and `onClick` events, so if you create your own component and do not explicitly add these mouse and pointer events as props, the tooltip will never appear. (See "Wrap custom component" example.)
```

Comment:
`Wrap custom component` should be a link.

Reply:
@afc163 , I had intended to do this, but I couldn't find an example of what the relative URL would be for an anchor link to an example. I took a guess and committed it, but please verify. Thanks!